spotlight/losses.py

```diff
@@ -116,22 +157,22 @@ def hinge_loss(positive_predictions, negative_predictions, mask=None):
                        positive_predictions +
                        1.0, 0.0)
 
-    if mask is not None:
-        mask = mask.float()
-        loss = loss * mask
-        return loss.sum() / mask.sum()
+    if sample_weights is not None or mask is not None:
```

Comment:
We don't need these checks here: we can always call the base loss function. I think that will make for less code repetition in all the loss functions.

Reply:
totally